templates/section.html

```diff
@@ -0,0 +1,9 @@
+<!--
+ templateType: section
+ label: Section
+ description: "A section that can be used to quickly scaffold out templates."
+ isAvailableForNewContent: false
+-->
+{% dnd_section %}
+
+{% end_dnd_section %}
```

Comment:
nit: line break

Reply:
This is also present in https://github.com/HubSpot/cms-sample-assets/blob/main/templates/partial.html and https://github.com/HubSpot/cms-sample-assets/blob/main/templates/global-partial.html ! So I think NBD, people will add it themselves anyway when adding content